ci-operator/step-registry/nutanix/validate-credential-rotation/nutanix-validate-credential-rotation-commands.sh

```
@@ -0,0 +1,380 @@
+rm pull-secret
+popd
+
+echo "Extracted credentials requests:"
+ls -l "${CR_DIR}"
+
+echo "=== Step 3: Generate new credentials using ccoctl ==="
+# Disable tracing for credential file creation
+set +x
+
+# Create the new Nutanix credentials file
+cat > ${SHARED_DIR}/new-credentials <<EOF
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)